pyaedt/generic/compliance.py

```diff
@@ -28,6 +28,16 @@
 
 
 class VirtualCompliance:
+    """Virtual compliance class.
```

Comment:
We should have a decriptions with a verb that ends in "s" or "es". Maybe something like "Provides for ensuring virtual compliance." (Although I have to admit that is not very meaningful or descriptive, at least to me.)